src/Formatters/Currency.php

```diff
@@ -15,7 +15,7 @@ public function __construct($amount)
 
     public function __toString()
     {
-        return Cashier::usesCurrencySymbol().number_format($this->amount, 2);
+        return config('cashier.currency') . number_format($this->amount, 2);
```

Comment:
We need to get the currency symbol here, not just the code, so we can do:
```suggestion
        return Currencies::getSymbol(strtoupper(config('cashier.currency')), config('cashier.currency_locale'));
```

You will need to add the use statement as well: `use Symfony\Component\Intl\Currencies;`